components/org.wso2.carbon.identity.api.user.totp/org.wso2.carbon.identity.api.user.totp.v1/src/main/java/org/wso2/carbon/identity/rest/api/user/totp/v1/core/TOTPService.java

```diff
@@ -110,6 +110,39 @@ public TOTPSecretResponseDTO getSecretKey() {
         return totpResponseDTO;
     }
 
+    /**
+     * Get the value in verifySecretkey claim.
+     *
+     * @return The value of verifySecretkey claim.
+     */
+    private String getVerifySecretKey() {
```

Comment:
we can remove this function as it is no longer used

Reply:
Fixed